tests/next_tests/unit_tests/conftest.py

```diff
@@ -23,12 +24,22 @@
 from gt4py.next.iterator import ir as itir, pretty_parser, pretty_printer, runtime, transforms
 from gt4py.next.program_processors import processor_interface as ppi
 from gt4py.next.program_processors.formatters import gtfn, lisp, type_check
-from gt4py.next.program_processors.runners import (
-    dace_iterator,
-    double_roundtrip,
-    gtfn_cpu,
-    roundtrip,
-)
+from gt4py.next.program_processors.runners import double_roundtrip, gtfn_cpu, roundtrip
+
+
+try:
+    import dace
+
+    # import dace_iterator only if dace is available
+    from gt4py.next.program_processors.runners import dace_iterator
+except ModuleNotFoundError as e:
+    if "dace" in str(e):
+        dace_iterator = None
+    else:
+        raise e
+finally:
+    if dace_iterator:
+        del dace
```

Comment:
If the explicit `import dace` above is removed, this part should also be removed